src/components/templates/Search/filterPrice.tsx

```diff
@@ -57,41 +58,96 @@ export default function FilterPrice({
   return (
     <div>
       <div className={styles.filterList}>
-        {filterItemsType.map((e, index) => {
-          const filter = cx({
-            [styles.selected]: e.value === serviceType,
+        {priceFilterItems.map((e, index) => {
+          const isSelected =
+            e.value === priceType || priceSelections.includes(e.value)
+          const selectFilter = cx({
+            [styles.selected]: isSelected,
             [styles.filter]: true
           })
           return (
             <Button
               size="small"
               style="text"
               key={index}
-              className={filter}
-              onClick={async () => {
-                await applyTypeFilter(e.value)
-              }}
+              className={selectFilter}
+              onClick={
+                // only works for two price options
+                isSelected
+                  ? async () => {
```

Comment:
good for initial prototype but we should try moving all this logic into ideally just one `handleFilterSelection()` method to which we can pass the `isSelected` as a parameter. So we can then do all logic within that handler method instead of repeating them

Reply:
I'm refactoring that right now. I'll push the new version when ready